application/src/main/resources/thingsboard.yml

```diff
@@ -612,6 +615,9 @@ js:
     max_errors: "${REMOTE_JS_SANDBOX_MAX_ERRORS:3}"
     # Maximum time in seconds for black listed function to stay in the list.
     max_black_list_duration_sec: "${REMOTE_JS_SANDBOX_MAX_BLACKLIST_DURATION_SEC:60}"
+    max_total_args_size: "${REMOTE_JS_SANDBOX_MAX_TOTAL_ARGS_SIZE:100000}"
```

Comment:
Please refactor these changes to save some "disk space". Let's avoid duplication of these params and put them on a higher level. For example: "js.max_total_args_size: ${JS_MAX_TOTAL_ARGS_SIZE:100000}